src/AlgebraicGeometry/Schemes/ProjectiveVariety/Objects/Methods.jl

```diff
@@ -15,7 +15,7 @@ end
 # one line printing
 function Base.show(io::IO, X::AbsProjectiveVariety{<:Field,<:MPolyQuoRing})
   if get(io, :supercompact, false)
-    print(io, "Scheme")
+    print(io, "Projective variety")
```

Comment:
Else case is missing this prefix and instead prints "V(...)" with "V" hardcoded that seems not good?

Reply:
Here was one of the solution to obtain explicit short printing, that could also fit well in nested printings. For a projective variety, we know the defining ideal `I` and it is common to use the notation `V(I)` to describe such objects. This is why the "V" is hardcoded, and then we print the ideal in between parentheses.